### ISSUE TITLE
Is there any way to cancel the swipe?

### ISSUE THREAD

**Issue by rodrigojuarez** [alexrainman/CarouselView#574]
I want to do some validation when the user starts to swipe and then have the possibility to cancel the swipe.
Is there any way I can do that?

**Comment by GeoSaffer** [comment]
I am looking for the same/similar functionality.

To only allow progressing forward in carousel when conditions are met.

Please, this would be great help.

**Comment by alexrainman** [comment]
Have you tried using the IsSwipeEnabled property?

**Comment by rodrigojuarez** [comment]
Yes, but if I set it to false, I can't do the validation when the user starts to swipe (because the user can't swipe), and if it's set to true, I don't see how to cancel the swipe if the validation fail.
I guess I'm looking for some sort of hook or event, SwipeStarted that can be cancelled.
